src/tests/Microsoft.Diagnostics.NETCore.Client/DARCVersion.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Consider adding this file to a .gitignore so that when we build it locally and commit that the file doesn't get included in the changes.

Alternatively, this should be template file somewhere not under any particular project and this project should:
1. Copy the file to the obj directory as a temporary file (e.g. DARCVersion.g.cs.tmp)
1. Find/replace the value
1. Compare it the existing file in the obj directory (e.g. compare DARCVersion.g.cs.tmp to DARCVersion.g.cs)
1. If different, then overwrite (overwrite DARCVersion.g.cs with DARCVersion.g.cs.tmp)
1. Dynamically include the file in the build.
This will prevent accidental overwrites in the source (because it's not under the project anymore) and will allow incremental build to continue working (your current solution will cause the test project to always build).